### PR TITLE
Update XCM/xtoken precompiles

### PR DESCRIPTION
**Pull request by PierreOssun**

### CODE REVIEW

**Review by gluneau — APPROVED** [review]
LGTM, with a commit.

File: docs/build/EVM/precompiles/xcm/native-transfer.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    function transfer_native() external payable {
+```
+
+#### 2. Asset Multilocation
+
+The Native token Multilocation is defined by: `Multilocation: { parents: 0, interior: Here }`
+The interior field is an empty bytes array (equivalent of `Here`).
+
+```solidity
+bytes[] memory interior1 = new bytes[](0);
```
Comment (gitofdeepanshu) [inline docs/build/EVM/precompiles/xcm/native-transfer.md:22]: have you tested all of these functions?

Comment (PierreOssun) [reply docs/build/EVM/precompiles/xcm/native-transfer.md:22]: YEs ofc. here: https://github.com/AstarNetwork/EVM-XCM-Examples/pull/2

File: docs/build/EVM/precompiles/xcm/xcm.md [hunk 2]
```diff
@@ -234,3 +238,11 @@ XCM.Multilocation memory asset = XCM.Multilocation({
     interior: interior
 });
 ```
+
+#### Builder Guides
+
+Three builder guides ont eh subject of EVM XCM are available in Builder section:
```
Comment (gitofdeepanshu) [inline docs/build/EVM/precompiles/xcm/xcm.md:244]: typo: On the*

**Pull request merged**